Tests/utils.lisp

```diff
@@ -17,149 +17,41 @@
 
 ;;; Line splitting utility
 ;;; ============================================================================
-
```

Comment:
I would prefer squashing the previous commit and this one into one new commit.

Also, the coverage achieved by this test seems insufficient: https://techfak.de/~jmoringe/9679c6b9cf9e6c559fa1ab6e1531f421.html

Reply:
squashed. as of coverage, it is hard to add by hand tests to it. fwiw it is also tested in more elaborate way in seos baseline regression test and in misc-tests.